osu.Game/Screens/Select/BeatmapDetails.cs

```diff
@@ -61,7 +61,7 @@ public BeatmapInfo Beatmap
                 approachRate.Value = beatmap.Difficulty.ApproachRate;
                 stars.Value = (float)beatmap.StarDifficulty;
 
-                if (beatmap.Metric?.Ratings.Count() > 0)
+                if (beatmap.Metric?.Ratings.Any() == true)

```

Comment:
it isn't because of `Metric?` `Any()` doesnt return a `bool` but a `bool?` which if doesn't like